guard/src/migrate/parser_tests.rs

```diff
@@ -392,6 +393,65 @@ fn test_parse_rules_file_rule_error() {
     );
 }
 
+#[test]
+fn test_disjunction_basic_clauses() {
```

Comment:
Please add tests for Conditional Rules and disjunction combinations across them as well.